app/feature/feature-addon-purchase/src/main/kotlin/com/hedvig/android/feature/addon/purchase/navigation/AddonPurchaseNavGraph.kt

```diff
@@ -72,6 +77,10 @@ fun NavGraphBuilder.addonPurchaseNavGraph(navigator: Navigator, navController: N
         navigateToSummary = { summaryParameters: SummaryParameters ->
           navigator.navigateUnsafe(Summary(summaryParameters))
         },
+        onNavigateToNewConversation = {
+          navController.typedPopBackStack<AddonPurchaseGraphDestination>(inclusive = true) // todo: is this the right thing?
```

Comment:
This looks correct to me, can you think of cases where we don't want this to happen?